tests/app/storage/test_encrypted_questionnaire_storage.py

```diff
@@ -37,7 +41,19 @@ def test_store_and_get(self):
         encrypted.save(data)
         # check we can decrypt the data
         self.assertEqual(
-            ("test", QuestionnaireStore.LATEST_VERSION), encrypted.get_user_data()
+            ("test", QuestionnaireStore.LATEST_VERSION, None), encrypted.get_user_data()
+        )
+
+    @freeze_time(datetime.now(tzutc()).replace(second=0, microsecond=0))
+    def test_store_and_get_submitted_at(self):
+        user_id = "1"
+        user_ik = "2"
+        encrypted = EncryptedQuestionnaireStorage(user_id, user_ik, "pepper")
+        data = "test"
+        encrypted.save(data, datetime.now(tz=tzutc()))
+        self.assertEqual(
+            ("test", QuestionnaireStore.LATEST_VERSION, datetime.now(tz=tzutc())),
+            encrypted.get_user_data(),
         )
```

Comment:
The vars don't provide any more value over keyword args especially since they are not reused. Plus, I think it makes it more readable.
```py
    def test_store_and_get_submitted_at(self):
        encrypted = EncryptedQuestionnaireStorage(
            user_id="1", user_ik="2", pepper="pepper"
        )
        encrypted.save(data="test", submitted_at=datetime.now(tz=tzutc()))

        self.assertEqual(
            ("test", QuestionnaireStore.LATEST_VERSION, datetime.now(tz=tzutc())),
            encrypted.get_user_data(),
        )
```

I think the test above should be cleaned up the same way and I think the tests need to be renamed to maybe something like:
```
- test_store_and_get_without_submiited_at
- test_store_and_get_with_submiited_at
```